Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org/'
 
-gem 'pmdtester', '~> 1.0.0.pre.beta3'
+gem 'pmdtester', :git => 'https://github.com/pmd/pmd-regression-tester.git'
```

Comment:
At some point, we should go back to a fixed version. But let's use the snapshot version for a while and test it that way.